### ISSUE TITLE
MongoDB

### ISSUE THREAD

**Issue by jordan2175** [stixworkinggroups/databaseschemas#1]
One of the things I would like to see is a basic implementation for something like MongoDB for Basic TTP -> Indicator -> Observable and maybe basic Course of Action.  My rational is, I believe this will be one of the most common use cases we will see in the wild.  And anything we can do to speed up adoption will be great.  

If we can get a series of schemas built and example database implementations done with best practices, that should really speed up adoption.  

Further, given the complex nature of STIX and its children, IMHO, most web developers will not understand all of the issues with building the backend database that will be needed to support it, this is where I think we come in.  If we can help them understand what is needed and even potentially provide some basic databases they can use, we could really speed up adoption.

**Comment by sparrell** [comment]
I agree with 'anything we can do to speed up adoption will be great.' and I agree mongodb is as good as any other nosql db to get going. Is there a (or should there be a) relation between the mongodb work and the json work? My gut is there is a linkage (or at least they shouldn't diverge) and the mongodb work will more closely parallel the json than the xml.
